script/Common.s.sol

```diff
@@ -13,5 +13,5 @@ struct EigenContracts {
 /// @dev Common script for all deployment scripts
 contract Common is Script {
     uint256 internal _privateKey =
-        vm.envOr("ANVIL_PRIVATE_KEY", uint256(0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80));
+        vm.envOr("PRIVATE_KEY", uint256(0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80));
```

Comment:
I bet this is the PRIVATE_KEY error I hit? ref revert https://github.com/Lay3rLabs/wavs-middleware/pull/98/commits/984307c97fe0404a725f6bd14cb1362f77655fc5 if so